jhrbva/Caravan
# Ca 9/signin functionality frontend

Functionality for the sign in page in front-end!
To test:
Try to log in with the user kcode and pass password
Check at the browser that you are redirected to /trips
You may or may not see the trip form depending on if that branch was merged already or not when you are testing
